_spinnaker/configure_ingress.md

```diff
@@ -1,162 +1,136 @@
 
 ```
-    alb.ingress.kubernetes.io/certificate-arn: arn:aws:acm:us-west-2:123456789012:certificate/abcdef01-1234-abcd-ef01-a1234567890f
+  hal config security api edit --override-base-url https://gate.demo.armory.io
+  hal config security ui edit --override-base-url https://demo.armory.io
+  hal deploy apply
 ```
 
-After this change is made, Spinnaker will no longer respond on port 80;
-HTTPS will be required.
 
-### Update the Internal URLS in Spinnaker (again!)
+## Exposing Spinnaker on GKE with Ingress
+### Setting up HTTP Load Balancing with Ingress
+
+GKE has a “built-in” ingress controller and that's what we will use.
 
-We'll need to update the internal URLs again (Deck will complain about trying
-to call out to an HTTP resource from an HTTPS request).  Update the URLs
-like we did before, but changing the protocols to `https`:
+First create a file called basic-ingress.yaml and paste it the following
 
 ```
-$ hal config security api edit --override-base-url https://gate.demo.armory.io
-$ hal config security ui edit --override-base-url https://demo.armory.io
-$ hal deploy apply
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: basic-ingress
+
+spec:
+  rules:
+  - host: demo.armory.io
+    http:
+      paths:
+      - backend:
+          serviceName: spin-deck
+          servicePort: 9000
+        path: /
+  - host: gate.demo.armory.io
+    http:
+      paths:
+      - backend:
+          serviceName: spin-gate
+          servicePort: 8084
+        path: /
 ```
 
-You should now be able to access Spinnaker via HTTPS.
+Then apply this
+`kubectl apply -f basic-ingress.yaml`
 
+Find out the external IP address of the load balancer serving your application by running:
+`kubectl get ingress basic-ingress`
 
-## Troubleshooting / Common Problems
 
-#### Connection Times Out
+Output:
+
+```
+NAME            HOSTS                                       ADDRESS         PORTS     AGE
+basic-ingress   demo.armory.io, gate.demo.armory.io         203.0.113.12    80        2m
+```
 
-Check the security group you have configured to make sure it will accept
-connections on port 80 (or 443 if you're using HTTPS) from your IP (or
-`0.0.0.0` if you want to allow it to be accessible from the world).
+Note: It may take a few minutes for GKE to allocate an external IP address and set up forwarding rules until the load balancer is ready to serve your application. In the meanwhile, you may get errors such as HTTP 404 or HTTP 500 until the load balancer configuration is propagated across the globe.
 
+You need to update your DNS records to have the demo.armory.io host point to the IP address generated.
 
+After doing that you can visit http://demo.dev.armory.io:9000/ to view spinnaker.
 
+### Secure with SSL on GKE
+To enable SSL and configure your certificates you can follow this guide:
+`https://cloud.google.com/kubernetes-engine/docs/how-to/ingress-multi-ssl
```

Comment:
This link isn't linked, not sure if the \` was supposed to do something here, or if it's a typo.  If you want to make this a real link, you'll need to do `[display text](url)`.